packages/features/bookings/Booker/components/BookEventForm/BookingFields.tsx

```diff
@@ -133,6 +135,10 @@ export const BookingFields = ({
           });
         }
 
+        if (field.prefillChecked && searchParams && searchParams[field.name]) {
```

Comment:
FormBuilder logic must be contained within FormBuilder. This place is suitable only for logic that is specific to Booking Form fields.